docs/site/compatibility.md

```diff
@@ -8,19 +8,29 @@ nav_order: 3
 
 The following table shows the currently supported algorithms.  The goal is to expand this over time with support from the underlying RAPIDS cuML libraries.  If you would like support for a specific algorithm, please file a [git issue](https://github.com/NVIDIA/spark-rapids-ml/issues) to help us prioritize.
 
-| Spark ML Algorithm     | Python | Scala |
+| Supported Algorithms   | Python | Scala |
 | :--------------------- | :----: | :---: |
-| K-Means                |   √    |       |
+| CrossValidator         |   √    |       |
+| KMeans                 |   √    |       |
 | k-NN (*)               |   √    |       |
 | LinearRegression       |   √    |       |
+| LogisticRegression     |   √    |       | 
 | PCA                    |   √    |   √   |
 | RandomForestClassifier |   √    |       |
 | RandomForestRegressor  |   √    |       |
+| UMAP (*)               |   √    |       |
+
+Note: Spark does not provide a k-Nearest Neighbors (k-NN) implementation, but it does have an [LSH-based Approximate Nearest Neighbor](https://spark.apache.org/docs/latest/ml-features.html#approximate-nearest-neighbor-search) implementation.   As an alternative to PCA, we also provide a Spark API for GPU accelerated Uniform Manifold Approximation and Projection (UMAP), a non-linear dimensionality reduction algorithm in the RAPIDS cuML library.
```

Comment:
UMAP docs already exist in the 23.08 branch, so maybe a rebase issue?  Presumably would merge OK, since it's a copy of existing text.

Reply:
In this file?   The top level README.md has it, but this file was overlooked.